src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -9,10 +13,13 @@
 import org.broadinstitute.hellbender.utils.read.CigarUtils;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
+import java.io.Serializable;
 import java.util.List;
 import java.util.Objects;
 
-class AlignmentRegion {
+@DefaultSerializer(AlignmentRegion.Serializer.class)
+class AlignmentRegion implements Serializable {
```

Comment:
I'd recommend removing `implements Serializable` and the `serialVersionUID` on the next line.  Since we intend to use Kryo serialization exclusively, mixing in Java serializability just confuses the issue.